src/main/java/com/codeborne/selenide/drivercommands/WebDriverWrapper.java

```diff
@@ -5,13 +5,20 @@
 import com.codeborne.selenide.Driver;
 import com.codeborne.selenide.proxy.SelenideProxyServer;
 import org.openqa.selenium.WebDriver;
+import org.testng.internal.Nullable;
 
+import javax.annotation.Nonnull;
+
+// TODO Remove me?
 public class WebDriverWrapper implements Driver {
   private final Config config;
   private final WebDriver webDriver;
   private final SelenideProxyServer selenideProxy;
 
-  public WebDriverWrapper(Config config, WebDriver webDriver, SelenideProxyServer selenideProxy) {
+  public WebDriverWrapper(@Nonnull Config config, @Nonnull WebDriver webDriver, @Nullable SelenideProxyServer selenideProxy) {
+    if (config == null) throw new IllegalArgumentException("config must not be null");
```

Comment:
You can easily use `Objects.requireNonNull` with explanation message in such case